src/service/chat.js

```diff
@@ -92,7 +133,6 @@ const sendSignedDMOtherProfile = async ({user_id, message}) => {
   };
 
   const response = await api.post('/chat/init-chat', payload);
-
   if (response.status === 200) {
     return Promise.resolve(response.data?.data);
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `response.status === 200` check is not necessary. If the status code is not in the 2xx range, the `axios` library will throw an error which can be caught and handled. Also, it's better to use a more descriptive variable name than `response`.

```diff
-    const response = await api.post('/chat/init-chat', payload);
-    if (response.status === 200) {
-      return Promise.resolve(response.data?.data);
-    }
+    const chatInitResponse = await api.post('/chat/init-chat', payload);
+    return Promise.resolve(chatInitResponse.data?.data);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->